pkg/addons/translate.go

```diff
@@ -606,7 +606,7 @@ var Translations = map[string][]struct {
 	"Secret/default/router-certs": {
 		{
 			Path:     jsonpath.MustCompile("$.stringData.'tls.crt'"),
-			Template: "{{ String (CertAsBytes .Config.Certificates.Router.Cert) }}\n{{ String (CertAsBytes .Config.Certificates.Ca.Cert) }}\n{{ String (PrivateKeyAsBytes .Config.Certificates.Router.Key) }}",
+			Template: "{{ String (CertChainAsBytes .Config.Certificates.Router.Certs) }}\n{{ String (CertAsBytes .Config.Certificates.Ca.Cert) }}\n{{ String (PrivateKeyAsBytes .Config.Certificates.Router.Key) }}",
```

Comment:
shouldn't need .Config.Certificates.Ca.Cert here any more?

Reply:
I dont know. Need to test if we dont have some unpredicted fallouts.

Reply:
As Router certificate will not have anything from internal CA. So if we want to trust it, we still need